CHANGELOG.md

```diff
@@ -1,25 +1,43 @@
 # Unreleased
 
-## Features
+## Breaking changes 
 
-### Preserve query strings
+This release ensures the GOV.UK Prototype Kit reflects the latest release of the GOV.UK Frontend, v4.0.0.
 
-We now preserve URL query strings when redirecting POST requests to GET requests.
+### Update to GOV.UK Frontend v4.0.0
 
-This means if you have a query like `/link/to/something?query=true&hello=world` on your POST form action, and you submit the form, the URL query string will be present in the redirected URL.
+The new release of GOV.UK Frontend contains:
+- an iteration to the accordion component
+- guidance about other ‘breaking’ changes you should make to improve your service
```

Comment:
This feels weird following the lead in sentence – the release doesn't contain guidance, it contains the changes?

```suggestion
- other ‘breaking’ changes you should make to improve your service
```